### PR TITLE
adding Azure-Samples/azure-edge-extensions-lnm-mq-isa95

### PR DESCRIPTION
**Pull request by bindsi**
The Azure-Samples/azure-edge-extensions-lnm-mq-isa95 defines two levels of an ISA-95 network model and enables MQTT bridging between the layers for bidirectional communication. This sample also showcases how to implement a Kafka Connector to retrieve data from a cloud-based Kafka Broker

### CODE REVIEW

File: drops/azure_edge_extensions_lnm_mq_isa95.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "Title": "Azure Edge Extensions AIO ISA-95 LNM and MQ",
+    "Summary": "This is a solution to build an ISA-95 network model on top of Azure IoT Operations utilizing AIO LNM and AIO MQ.",
```
Comment (fcabrera23) [inline drops/azure_edge_extensions_lnm_mq_isa95.json:3]: Suggestion -> Add some extra wording about what "AIO" and LNM and MQ stands for - Probably try to use full names, as people reading the content maybe they're not familiar

File: drops/azure_edge_extensions_lnm_mq_isa95.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
+{
+    "Title": "Azure Edge Extensions AIO ISA-95 LNM and MQ",
```
Comment (fcabrera23) [inline drops/azure_edge_extensions_lnm_mq_isa95.json:2]: Maybe try to define the goal here -> Is this an automated delpoyment?

File: drops/azure_edge_extensions_lnm_mq_isa95.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "YAML"
+    ],
+    "Products": [
+        "Azure IoT Operations",
+        "Arc",
+        "KafkaConnector"
+    ],
+    "LastModified": "2024-07-04T00:00:00.000Z",
+    "CreatedDate": "2024-07-04T00:00:00.000Z",
+    "Topics": []
```
Comment (fcabrera23) [inline drops/azure_edge_extensions_lnm_mq_isa95.json:29]: Maybe add Purdue model? ISA-95? Industrail newortking? thigs that can help the user to search your drop

File: drops/azure_edge_extensions_lnm_mq_isa95.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    "Source": "https://github.com/Azure-Samples/azure-edge-extensions-lnm-mq-isa95",
+    "Type": "sample_app",
+    "Difficulty": "Medium",
+    "ProgrammingLanguage": [
+        "YAML"
+    ],
+    "Products": [
+        "Azure IoT Operations",
+        "Arc",
+        "KafkaConnector"
```
Comment (fcabrera23) [inline drops/azure_edge_extensions_lnm_mq_isa95.json:25]: Missing space? "Kafka Connector?
What about MQ? ISA-95?

**Review by fcabrera23 — CHANGES_REQUESTED** [review]
@bindsi thanks for your contributions - Please check my comments

File: drops/azure_edge_extensions_lnm_mq_isa95.json (hunk 1 above)
Comment (bindsi) [reply drops/azure_edge_extensions_lnm_mq_isa95.json]: agree

File: drops/azure_edge_extensions_lnm_mq_isa95.json (hunk 2 above)
Comment (bindsi) [reply drops/azure_edge_extensions_lnm_mq_isa95.json]: done and also avoided to use the acronyms, except for MQ for is the commonly used wording for "Message Queueing"

File: drops/azure_edge_extensions_lnm_mq_isa95.json (hunk 4 above)
Comment (bindsi) [reply drops/azure_edge_extensions_lnm_mq_isa95.json:26]: agree...KafkaConnector is one single word used in tech...also our AIO docs use it as single word:
[https://learn.microsoft.com/en-us/azure/iot-operations/connect-to-cloud/howto-configure-kafka?tabs=managed-identity#kafkaconnector](https://learn.microsoft.com/en-us/azure/iot-operations/connect-to-cloud/howto-configure-kafka?tabs=managed-identity#kafkaconnector)

**Review by fcabrera23 — APPROVED** [review]
LGTM

**Pull request merged**